tools/myst/enc/enc.c

```diff
@@ -37,6 +37,13 @@
 #define IRETFRAME_EFlags IRETFRAME_SegCs + 8
 #define IRETFRAME_Rsp IRETFRAME_EFlags + 8
 
+static myst_kernel_args_t kargs;
+
+long myst_syscall(long n, long params[6])
```

Comment:
This syscall is unique in that the caller is from a layer below the kernel. Suggest rename this function to differentiate from the other `myst_syscall` that is originated from libc.